packages/flagging/src/evaluation.spec.ts

```diff
@@ -31,6 +31,9 @@ describe('evaluate', () => {
     expect(result).toEqual({
       value: true,
       variant: 'variation-124',
+      flagMetadata: {
```

Comment:
plumbing the `allocationKey` through the metadata so that we have it available when tracking exposures